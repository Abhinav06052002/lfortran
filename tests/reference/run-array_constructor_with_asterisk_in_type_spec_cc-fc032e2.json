{
    "basename": "run-array_constructor_with_asterisk_in_type_spec_cc-fc032e2",
    "cmd": "lfortran --continue-compilation --no-color {infile}",
    "infile": "tests/errors/array_constructor_with_asterisk_in_type_spec_cc.f90",
    "infile_hash": "2a778b4f9d0ee13f1a1903f21829fc4454129926f2e3e085445ce6d5",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "run-array_constructor_with_asterisk_in_type_spec_cc-fc032e2.stdout",
    "stdout_hash": "d95a84435a0b0d3840e24be60f20b0a5d37007f01c9bec5e1b1becab",
    "stderr": "run-array_constructor_with_asterisk_in_type_spec_cc-fc032e2.stderr",
    "stderr_hash": "de07e654a150f771b251bdd517900d261b4746fadb1955f2a0a1ba7c",
    "returncode": 1
}