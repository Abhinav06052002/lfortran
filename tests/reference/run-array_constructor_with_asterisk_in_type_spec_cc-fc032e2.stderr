semantic error: Type-spec cannot contain an asterisk for a type parameter
